utils/runtime.go

```diff
@@ -59,3 +68,8 @@ func CheckRuntime(ctx context.Context, runtime *sharedpb.SignedRuntimeInfoReques
 	// Reserve: only pass fatal error to higher level
 	return report, nil
 }
+
+func WhoAmI() string {
+	_, en := env.GetEnv(WHOAMI_ENV_NAME)
```

Comment:
The environment does not control which service to call. For example, I can be guard service, but I am *calling* escrow service, so the issue is that `address` determines dial timeout along with who it is.

Reply:
> The environment does not control which service to call. For example, I can be guard service, but I am _calling_ escrow service, so the issue is that `address` determines dial timeout along with who it is.

Got it. The timeout depends on server side rather than the client side. Will do it.